packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -0,0 +1,415 @@
+                                                )}
+                                            </Text>
+                                        )}
+                                        <FormSubHeader title={localize('Address')} title_text_size='s' />
+                                        <PersonalDetailsForm
+                                            is_qualified_for_poa
+                                            errors={errors}
+                                            touched={touched}
+                                            values={values}
+                                            handleChange={handleChange}
+                                            handleBlur={handleBlur}
+                                            setFieldValue={setFieldValue}
+                                            setFieldTouched={setFieldTouched}
+                                            editable_fields={changeable_fields}
+                                            states_list={states_list}
+                                        />
+                                        <FormSubHeader title={localize('Document submission')} title_text_size='s' />
+                                        <FormBodySection>
+                                            <FileUploaderContainer
+                                                onRef={ref => (file_uploader_ref = ref)}
+                                                onFileDrop={df => {
+                                                    setDocumentFile({
+                                                        files: df.files,
+                                                        error_message: df.error_message,
+                                                    });
+                                                }}
+                                                getSocket={WS.getSocket}
+                                                files_description={<FilesDescription />}
+                                                examples={<CommonMistakeExamples />}
+                                            />
+                                        </FormBodySection>
+                                    </FormBody>
+                                </ThemedScrollbars>
+                                {is_qualified_for_cfd_modal ? (
+                                    <Modal.Footer has_separator>
+                                        <FormSubmitButton
+                                            is_disabled={
+                                                isSubmitting ||
+                                                !!isFieldsIssue(errors, values) ||
+                                                (document_file.files && document_file.files.length < 1) ||
+                                                !!document_file.error_message
+                                            }
+                                            label={localize('Save and submit')}
```

Comment:
Is this FormSubmitButton for MT5 poa modal?

Reply:
if yes, are we changing the cta-label for POA?

Reply:
did we change the design for CFD-POA?

Reply:
The current cfd-poa modal  in production does'nt have previous button and sub,it button is different.

Reply:
@amina-deriv , yes. there is figma link inside one of the subcards

Reply:
But what is expected on clicking previous button

Reply:
According to the card requirements, the client will be navigated to jurisdiction modal

Reply:
Based on the code, it seems that when the user clicks the "Back" button, they are redirected to the jurisdiction modal.
However, from what I remember, the CFD-modal can be accessed from both the "Compare Accounts" table and the jurisdiction table.
Considering this, if a user is moving to the CFD-POA modal from the "Compare Accounts" table, wouldn't clicking the "Back" button open the jurisdiction modal instead? 

Please help to clarify the expected behaviour here.